src/js/components/Pagination/doc.js

```diff
@@ -92,105 +92,96 @@ export const themeDoc = {
     the pagination controls.`,
     type: 'string | (props) => {}',
   },
-  'pagination.control.extend': {
-    description: `Any additional style for each control.`,
-    type: 'string | (props) => {}',
-  },
-  'pagination.control.pad': {
-    description: `Padding around each control's label.`,
-    type: 'string | object',
-    defaultValue: '4px',
-  },
-  'pagination.control.size.small.border.radius': {
-    description: `Rounding of the corners for each control.`,
+  'pagination.button.size.small.border.radius': {
```

Comment:
Can we move these to be alphabetized (e.g. move the pagination.containers to follow afterward)?

Also, since we are now supporting all button props, can we now have a single `pagination.button` which is treated similarly to Box or Drop referencing "any valid Button props to specify the pagination button styling"?

Reply:
Yes, I will update.

Reply:
Done.